landscape/explanation/glossary/ppa.md

```diff
@@ -0,0 +1,6 @@
+# PPA 
+
+**Personal Package Archive(PPA)** is a software repository specifically made for Ubuntu-based products. This tool allows developers to create their own repositories that they can use to distribute their own software. In the context of Landscape, the PPA contains the tool's software for the server. 
+When installing and deploying the Landscape server on one machine, it is important to be mindful of the PPA being used as the [Quickstart installation](https://ubuntu.com/landscape/docs/quickstart-deployment) guide mentions that it varies upon the version of server being used. For example, if the beta version of Landscape is being used, [its PPA is `ppa:landscape/self-hosted-beta`](https://ubuntu.com/landscape/docs/quickstart-deployment) whereas the [recent server's PPA is: `ppa:landscape/self-hosted-23.03`](https://ubuntu.com/landscape/docs/quickstart-deployment).  
+
+Think of Landscape's PPA as picking a movie or show from your playlist on your favorite streaming site. You get to pick specific software from the PPA based on the needs of your software's repository. 
```

Comment:
```suggestion

```

Reply:
I decided to remove this analogy because I don't think one is necessary here with all the other helpful info you've included. :smiley: 

But also (and this is only because I know Landscape well), I don't think this analogy is quite accurate - I think a more accurate analogy with your context would be that Landscape PPA is a streaming service (e.g., Amazon Prime Video) where you want to watch exclusive content (e.g., An Amazon Prime Video-only show, such as Fallout). So, in order to get the Landscape software (Fallout), you need to first have its location (Amazon).

That's just for your own knowledge though! No need to refactor that into an analogy for the documentation.